nextflow_schema.json

```diff
@@ -586,7 +523,7 @@
                 },
                 "gtdbtk_pplacer_cpus": {
                     "type": "number",
```

Comment:
why not use integers to count CPUs?

Reply:
Fixed! I dunno why it was a number... might be very old left over